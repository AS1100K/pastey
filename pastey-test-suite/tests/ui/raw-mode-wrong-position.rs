use pastey::paste;

macro_rules! m {
    ($name:ident) => {
        paste! {
            struct [< $name:camel # >];
        }
    };
}

m!(r#loop);

fn main() {}
