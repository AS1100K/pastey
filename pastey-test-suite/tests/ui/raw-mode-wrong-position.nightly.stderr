error: `#` is reserved keyword and it enables the raw mode (i.e. generate Raw Identifiers) and it is only allowed in the beginning like `[< # ... >]`
  --> tests/ui/raw-mode-wrong-position.rs:6:35
   |
 6 |             struct [< $name:camel # >];
   |                                   ^
...
11 | m!(r#loop);
   | ---------- in this macro invocation
   |
   = note: this error originates in the macro `m` (in Nightly builds, run with -Z macro-backtrace for more info)
